### PR TITLE
add gsl to bcr

### PR DESCRIPTION
**Pull request by shabanzd**
added gsl to bcr

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (gsl) have been updated in this PR. Please review the changes.

File: modules/gsl/4.0.0/source.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "integrity": "sha256-65H8sQpqpcyx0iTgelbI7P/pobtgH6GEgnbsRqIgC/s=",
```
Comment (meteorcloudy) [inline modules/gsl/4.0.0/source.json:2]: Please add patch_strip = 0 in source.json

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/gsl/4.0.0/source.json (hunk 1 above)
Comment (shabanzd) [reply modules/gsl/4.0.0/source.json:2]: done :)

File: modules/gsl/metadata.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "versions": [
```
Comment (meteorcloudy) [inline modules/gsl/metadata.json:2]: Hmm, just noticed the information in metadata.json isn't complete. Can you add missing fields?

for example, https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/abseil-cpp/metadata.json

I'm a bit embarrassed that we don't have validation for this.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Review by Wyverald — APPROVED** [review]

**Pull request merged**